app/src/main/res/layout/activity_confirm_overtime.xml

```diff
@@ -15,18 +16,272 @@
+                                >
+                                <TextView
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:text="@string/project_name"
+                                    android:textColor="@color/colorPrimary"
+                                    />
+                                <TextView
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:textSize="@dimen/sp_18"
+                                    />
+                            </LinearLayout>
+                        </TableRow>
+                        <TableRow
+                            android:layout_marginTop="@dimen/dp_10"
+                            android:gravity="center_vertical"
+                            >
+                            <ImageView
+                                android:layout_width="@dimen/dp_30"
+                                android:layout_height="@dimen/dp_30"
+                                android:tint="@color/colorPrimary"
+                                app:srcCompat="@drawable/ic_clock"
+                                />
+                            <LinearLayout
+                                android:layout_marginLeft="@dimen/dp_20"
+                                android:orientation="vertical"
+                                >
+                                <TextView
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:text="@string/from"
+                                    android:textColor="@color/colorPrimary"
+                                    />
+                                <TextView
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:textSize="@dimen/sp_18"
+                                    />
+                            </LinearLayout>
+                        </TableRow>
+                        <TableRow
+                            android:layout_marginTop="@dimen/dp_10"
+                            android:gravity="center_vertical"
+                            >
+                            <ImageView
+                                android:layout_width="@dimen/dp_30"
+                                android:layout_height="@dimen/dp_30"
+                                android:tint="@color/colorPrimary"
+                                app:srcCompat="@drawable/ic_clock"
+                                />
+                            <LinearLayout
+                                android:layout_marginLeft="@dimen/dp_20"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:layout_marginStart="@dimen/dp_20"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)